bindings/matrix-sdk-ffi/src/client_builder.rs

```diff
@@ -208,7 +208,7 @@ impl Default for ClientBuilder {
             sliding_sync_proxy: None,
             proxy: None,
             disable_ssl_verification: false,
-            inner: MatrixClient::builder(),
+            inner: MatrixClient::builder().handle_refresh_tokens(),
```

Comment:
Could this be exposed as a public setter on the `ClientBuilder` instead?

Reply:
Sure, would it be ok to expose it as a bool though with the value defaulted to `true`? I don't think there are common cases when you wouldn't want to refresh the tokens automatically, and from the FFI level I don't think enough of the inner client are exposed to be able to deal with an expired token so it wouldn't be much use anyway.

Reply:
I think in a multiprocess setup, you would only want one process to handle token refreshing, right?

Reply:
I wish it was that simple, but the app can be running without any extensions and equally an extension can be running without the app. They all need to be able to refresh the token if it has expired with a multi-process lock to fix the case where 2 are running at the same time.